pkg/ccl/backupccl/backuppb/backup.proto

```diff
@@ -55,6 +55,8 @@ message BackupManifest {
     // ApproximatePhysicalSize is the approximate size of the physical bytes in
     // compressed SST form.
     uint64 approximate_physical_size = 11;
+
+    bool has_range_keys = 12;
```

Comment:
huh, so, on further pondering, given this bool's zero value, I think we need to condition inc support on the backup version being strictly greater than v24_1.

We _could_ flip this to contains_no_range_keys so that the zero value is to assume it _does_ contain them, but given RKs should be pretty rare I kinda prefer the current zero value and think we can just filter len(mainfests) > 1 ==> v > v24_1

Reply:
Good catch. Will do.

Reply:
added:

```
if manifests[0].ClusterVersion.Less(clusterversion.V24_2.Version()) && len(manifests) > 1 {
		return errors.Newf("cannot run online restore on incremental backups taken on %s or older", clusterversion.V24_1.String())
	}
```